install

```diff
@@ -4,7 +4,7 @@ cd $(dirname $0)
 
 # Compatibility logic for older Anaconda versions.
 if [ "${CONDA_EXE} " == " " ]; then
-    CONDA_EXE=$((find /opt/conda/bin/conda || find ~/anaconda3/bin/conda) 2>/dev/null)
+    CONDA_EXE=$((find /opt/conda/bin/conda || find ~/anaconda3/bin/conda || find ~/miniconda3/bin/conda) 2>/dev/null)
```

Comment:
There's a similar check in the `uninstall` script as well. Could you please add it too?

Reply:
Good idea, pushed.